src/test/java/com/puppycrawl/tools/checkstyle/internal/utils/XdocUtil.java

```diff
@@ -77,9 +80,7 @@ public static Set<Path> getXdocsConfigFilePaths(Set<Path> files) {
         final Set<Path> xdocs = new HashSet<>();
         for (Path entry : files) {
             final String fileName = entry.getFileName().toString();
-            if (fileName.startsWith("config_")) {
-                xdocs.add(entry);
-            }
+            xdocs.add(entry);
```

Comment:
I include all files. This will make the test a bit slower since we are now going through all xdoc files. If you have a better idea let me know. 

Maybe do pattern matching like below?
```java
            if (fileName.startsWith("config_")
|| see if entry matches pattern like "src/xdocs/.+/fileName" 
) {
```

Reply:
Add it to this PR, it can be as separate commit if that is convenient for you to keep it separate.